test/IntegrationTests/ContinuousProfilerTests.cs

```diff
@@ -17,6 +14,7 @@ public ContinuousProfilerTests(ITestOutputHelper output)
     {
     }
 
+#if NET // allocator tests are only supported on .NET
```

Comment:
Can we at least test that the application is not failing if env vars are set - just zero-code returning?